docker-setup/localstack/init.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
We've got some whitespace issue.

`ditto-localstack     | /docker-entrypoint-initaws.d/init.sh: line 2: $'\r': command not found`

This works okay. 
```
#!/bin/bash
aws configure set aws_access_key_id "XXX"
aws configure set aws_secret_access_key	"XXX"
aws configure set aws_session_token "XXX"
aws configure set region "eu-west-2"
aws configure set output "json"
aws --endpoint-url=http://localstack:4566 kinesis create-stream --stream-name ditto --shard-count 1
aws --endpoint-url=http://localstack:4566 kinesis wait stream-exists --stream-name ditto
```

Reply:
Very minor but you could use `awslocal`